docs/data-sources/search.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "akamai"
+page_title: "Akamai: akamai_search"
+subcategory: "Property Provisioning"
+description: |-
+ Search
+---
+
+# akamai_search
+
+
+Use the `akamai_search` data source to query and retrieve the list of properties that match a specific hostname, edge hostname or property 
+name based on the [EdgeGrid API client token](https://techdocs.akamai.com/developer/docs/authenticate-with-edgegrid) you're using.
+
+## Example usage
+
+Return properties associated with the EdgeGrid API client token currently used for authentication:
+
+
+```hcl
+datasource "akamai_search" "example" {
+  key = "hostname"
+  value = "test.akamai.com"
+}
+
+output "my_property_list" {
+  value = data.akamai_search.example
+}
+```
+
+## Argument reference
+
+This data source supports these arguments:
+
+* `key` - (Required) Key used for search. Valid values are `hostname`, `edgeHostname`, `propertyName`.
+* `value` - (Required) Value to search for.
+
+## Attributes reference
+
+This data source returns this attribute:
+
+* `properties` - A list of properties available for the contract and group IDs provided.
```

Comment:
```suggestion
* `properties` - A list of properties matching the given criteria.
```

Reply:
changed but moved to docs/data-sources/property_search.md